src/taskClass.dart

```diff
@@ -0,0 +1,18 @@
+class TaskListApp {
+  // TODO: Implement Task list app
+  int id = 0;
+  String title = "";
+  String description = "";
+  bool isCompleted = false;
+  static int count = 0;
+  static Set<TaskListApp> tasks = {};
```

Comment:
since the list is here, you could make functions to manipulate it directly within this app